pages/guide.py

```diff
@@ -259,7 +267,9 @@ def _get_field_val(self, field: str, field_type: str) -> Any:
       return self.form.get(field)
     elif field_type == 'split_str':
       val = self.split_input(field, delim=',')
-      if field == 'blink_components' and len(val) == 0:
+      if field == 'rollout_platforms':
+        val = self.form.getlist(field)
```

Comment:
I have question for my own understanding: why use self.form.getlist instead of [split_input](https://github.com/GoogleChrome/chromium-dashboard/blob/d331c4540eb0a9c37e091021f92c53942770f04c/framework/basehandlers.py#L457) that was already used above? It uses the form already too.

Reply:
this is because we use the select input here, and it stores data into an array and not a string separated list